static/data/team.json

```diff
@@ -21,15 +21,17 @@
     },
     {
       "name": "Sadra Khanjari",
-      "email": "",
-      "linkedin": "",
-      "github": ""
+      "email": "sadra.khanjari@gmail.com",
+      "linkedin": "https://www.linkedin.com/in/sadra-khanjari/",
+      "img": "https://avatars.githubusercontent.com/u/45975703?v=4",
+      "github": "https://github.com/SK2iP"
     },
     {
       "name": "Arad Firouzkouhi",
-      "email": "",
-      "linkedin": "",
-      "github": ""
+      "email": "aradfir@gmail.com",
```

Comment:
are you ARAD?

Reply:
Last night, Arad submitted a pull request with his profile, and I accepted it. Then, I pulled from main.